spec/features/component_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Component Page' do
```

Comment:
Is there a reason you wrote a feature test instead of component tests for this?

Reply:
Yes haha because I was having a hard time getting the component tests to work correctly (specifically passing the document/presenter/etc around). I would be happy to discuss a better way!

Reply:
We could pair on it if that would help. I know setting up the Blacklight objects for testing can be difficult.